bin/git-mr

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -eu
+
+if test "${1-}" = "clean"; then
+  git for-each-ref refs/heads/mr/* --format='%(refname)' | while read ref; do
+    git branch -D ${ref#refs/heads/}
+  done
+  exit 0
+elif [[ ${1-} =~ ^(https?://[^/]+/(.+))/merge_requests/([0-9]+).*$ ]]; then
+  remote=${BASH_REMATCH[1]}.git
+  id=${BASH_REMATCH[3]}
+  branch=mr/$id
+else
+  test -z "${1-}" && echo "mr number required." 1>&2 && exit 1
+  id=$1
+  remote=${2:-origin}
+  branch=mr/$id
+fi
+git fetch -fu $remote refs/merge-requests/$id/head:$branch && \
+git checkout $branch && \
+git config --local --replace branch.$branch.merge refs/mr/$id/head && \
```

Comment:
Hmm, actually, i use `mr` to be sync with `git-pr` which save PR heads in `pr/X`.

I updated thinks a bit. But i cant reproduce the bug.

``` console
$ ~/src/perso/git-extras/bin/git-mr 1
From gitlab.dalibo.info:clients/dalibo_socle
 * [new ref]         refs/merge-requests/1/head -> mr/1
Previous HEAD position was a77a459... Merge branch 'pgbouncer' into 'master'
Switched to branch 'mr/1'
$ git config --local branch.mr/1.merge
refs/merge-requests/1/head
$ git pull origin
From gitlab.dalibo.info:clients/dalibo_socle
 * branch            refs/merge-requests/1/head -> FETCH_HEAD
Already up-to-date.
$
```

Can you give me more details ?